Makefile

```diff
@@ -672,7 +672,7 @@ pkg/pb/frontend.pb.go: pkg/pb/messages.pb.go
 pkg/pb/matchfunction.pb.go: pkg/pb/messages.pb.go
 pkg/pb/mmlogic.pb.go: pkg/pb/messages.pb.go
 pkg/pb/evaluator.pb.go: pkg/pb/messages.pb.go
-internal/pb/synchronizer.pb.go: pkg/pb/messages.pb.go
+internal/ipb/synchronizer.pb.go: pkg/pb/messages.pb.go
```

Comment:
This change is good, but because you did too many things in one pr, no approval :(